<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFromFileButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image From File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matchBreed">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="pos1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="pos2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="pos3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Third Match</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loadFromFileButton</sender>
   <signal>clicked()</signal>
   <receiver>label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>matchBreed</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
